Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -93,6 +94,25 @@ void WebFoundTextRangeController::findTextRangesForStringMatches(const String& s
         foundTextRanges.append(foundTextRange);
     }
 
+#if ENABLE(PDF_PLUGIN)
+    for (RefPtr frame = &m_webPage->corePage()->mainFrame(); frame; frame = frame->tree().traverseNext()) {
+        RefPtr localFrame = dynamicDowncast<LocalFrame>(frame.get());
+        if (!localFrame)
+            continue;
+
+        if (RefPtr pluginView = WebPage::pluginViewForFrame(localFrame.get())) {
+            auto currentFrameName = frame->tree().uniqueName();
+            order++;
```

Comment:
what does this order variable mean?

Reply:
"frame order" would probably be more accurate – it's existing though, so I'm not going to rename it now.